EMRM.py

```diff
@@ -1,25 +1,40 @@
 import numpy as np
-import pandas as pd
 
 from scipy.integrate import quad
 from scipy.linalg import sqrtm
 from scipy.stats import entropy
 
 class Vocabulary:
-    def __init__(self, characters, wlen, input_file):
-        self.characters = characters
+    """ Store relevant language knowledge of all true words in a vocabulary.
+
+    Attributes:
+        characters: A string of all possible characters.
```

Comment:
the attribute (unlike the argument) is a set

Reply:
thanks, corrected in new version